app/stylesheets/SplitCost.scss

```diff
@@ -0,0 +1,12 @@
+.tilt-icon.tilt-icon-split-cost {
+  width: 23px;
+  height: 25px;
+  fill: white;
+}
+.outline {
+  stroke: #CFD6DC;
+  stroke-width: 2px;
+}
+.bottom-fold {
+  fill: #CFD6DC;
```

Comment:
ping - yeah, looks like this color is used in 9 different files now. Would you be willing to make a variable in _colors.scss